### PR TITLE
Fix parse tombstone

### PR DESCRIPTION
**Pull request by WeizhongX**
reuse the command line parameters that already exist,
call maybe_parse_tombstone from check_crash

### CODE REVIEW

**Comment by foolip** [comment]
The wpt-chrome-dev-* CI failures here were likely due to https://github.com/web-platform-tests/wpt/pull/19360. A workaround was landed in https://github.com/web-platform-tests/wpt/pull/28238, so I'm closing and reopening to re-trigger the CI checks.

File: tools/wptrunner/wptrunner/browsers/chrome_android.py [hunk 1]
```diff
@@ -189,15 +189,19 @@ def logcat_cmd(self):
         cmd.extend(['logcat', '*:D'])
         return cmd
 
-    def maybe_parse_tombstone(self, logger):
-        if self.stackparser_script:
-            cmd = [self.stackparser_script, "-a", "-w"]
+    def check_crash(self, process, test):
+        self.maybe_parse_tombstone()
+        return False
```
Comment (jgraham) [inline tools/wptrunner/wptrunner/browsers/chrome_android.py:194]: Can you add a comment here about why we're always returning `False`?

File: tools/wptrunner/wptrunner/browsers/chrome_android.py [hunk 2]
```diff
@@ -189,15 +189,21 @@ def logcat_cmd(self):
         cmd.extend(['logcat', '*:D'])
         return cmd
 
-    def maybe_parse_tombstone(self, logger):
-        if self.stackparser_script:
-            cmd = [self.stackparser_script, "-a", "-w"]
+    def check_crash(self, process, test):
+        self.maybe_parse_tombstone()
+        # Existance of a tombstone does not necesssary mean test target has
```
Comment (LukeZielinski) [inline tools/wptrunner/wptrunner/browsers/chrome_android.py:194]: nit: couple typos
Existence, and
necessarily

**Review by LukeZielinski — APPROVED** [review]

File: tools/wptrunner/wptrunner/browsers/chrome_android.py (hunk 2 above)
Comment (WeizhongX) [reply tools/wptrunner/wptrunner/browsers/chrome_android.py]: done

**Pull request merged**